torntools/scripts/content/faction/ttWarReport.js

```diff
@@ -0,0 +1,57 @@
+warReportLoaded().then(() => {
+	console.log("TT - Faction | War Report");
+
+	displayContainer();
+});
+
+function warReportLoaded() {
+	return requireElement("div.faction-war");
+}
+
+function displayContainer() {
+	let options_container = content.newContainer("War Report", { first: true, id: "ttWarReport" });
+
+	let export_btn = doc.new("div");
```

Comment:
It would be better to use the `doc.new` function for the other fields. This also applies to the element under this.

`doc.new({ type: "div", id: "ttExportTableButton", text: "Export Table to CSV" });`

Reply:
Done.